src/components/spatial/Spatial.js

```diff
@@ -383,16 +383,7 @@ class Spatial extends AbstractSpatialOrScatterplot {
   onUpdateCellsLayer() {
     const { layers } = this.props;
     const layerDef = (layers || []).find(layer => layer.type === 'cells');
-    const hasBitmaskLayers = (layers || []).find(
-      layer => layer.type === 'bitmask',
-    );
-    if (layerDef
-      && this.cellsEntries.length
-      // Only use cellsEntries in quadtree calculation if there is
-      // centroid data in the cells (i.e not just ids).
-      // eslint-disable-next-line no-unused-vars
-      && this.cellsEntries[0][1].xy
-      && !hasBitmaskLayers) {
+    if (layerDef) {
```

Comment:
This reverts back to what existed previously because we don't need to check for the cells layer anymore here as it is not passed in